tests/Doctrine/ODM/MongoDB/Tests/Aggregation/Stage/UnsetTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ODM\MongoDB\Tests\Aggregation\Stage;
+
+use Doctrine\ODM\MongoDB\Aggregation\Stage\UnsetStage;
+use Doctrine\ODM\MongoDB\Tests\Aggregation\AggregationTestTrait;
+use Doctrine\ODM\MongoDB\Tests\BaseTest;
+use Documents\User;
+
+class UnsetTest extends BaseTest
+{
+    use AggregationTestTrait;
+
+    public function testUnsetStage(): void
+    {
+        $documentPersister = $this->dm->getUnitOfWork()->getDocumentPersister(User::class);
+        $unsetStage        = new UnsetStage($this->getTestAggregationBuilder(), $documentPersister, 'id', 'foo', 'bar');
+
+        self::assertSame(['$unset' => ['_id', 'foo', 'bar']], $unsetStage->getExpression());
+    }
+
+    public function testLimitFromBuilder(): void
```

Comment:
```suggestion
    public function testUnsetFromBuilder(): void
```

Maybe you want to just rename all of these tests to "testFromBuilder()"?

Reply:
Good idea, that way people won't see just how much copy/paste my job consists of 😇